Automattic/wp-calypso
# Enable Gutenberg NUX with Gutenberg copy

#### Changes proposed in this Pull Request
1. Adds images from core to show in the NUX guide
2. Updates file to essentially match the latest changes from core
3. Use WordPress.com specific language (WordPress editor instead of block editor) and link to wpcom support pages.
4. Enable wpcom NUX

#### Testing instructions
0. Sandbox the public API, and two URLs for two test sites you are about to create
1. In a private window, go to wordpress.com/start and create a new test account.
2. Go through the flow, creating the first of the sites you've sandboxed.
3. Edit a page in the new site. It should display a NUX modal. You should see "WordPress editor" instead of "block editor" as the title of the tour:
<img width="767" alt="Screen Shot 2020-03-03 at 1 12 44 PM" src="https://user-images.githubusercontent.com/6265975/75820387-1f795180-5d51-11ea-85e3-c69f20d8f096.png">

4. Additionally, on the last page, the support link should take you to the wordpress.com editor support page:
<img width="1186" alt="Screen Shot 2020-03-03 at 1 12 54 PM" src="https://user-images.githubusercontent.com/6265975/75820428-2ef89a80-5d51-11ea-9a63-2ed9b08c9873.png">

5. Dismiss the modal.
6. Close the private window (effectively clearing the cookies and site data)
7. Reopen the private window and log into your new test account on wordpress.com
8. In the same site as above, edit a page. **The NUX tour should not display.**
9. In the Calypso sidebar, there is an option at the bottom to "Add New Site". Go through that flow to create a new site. Use the second URL you sandboxed at the beginning.
10. In that new site, when you edit a page, verify that **the NUX tour does not display**.
11. In the editor options, choose "welcome guide". 
<img width="389" alt="Screen Shot 2020-03-03 at 1 21 37 PM" src="https://user-images.githubusercontent.com/6265975/75820840-f60cf580-5d51-11ea-9f8a-65fcc264f0b0.png">

12. It should open. _Do not close the guide._ Instead, use the back button to return to Calypso. Go to a different site.
13. Verify that editing a page in that other site will now show the NUX as well.
13. Dismiss the NUX and it should no longer show on either site.

_I went through the above steps, and it is working correctly._

Resolves #39641 once it is deployed.
